edith-step/src/App.js

```diff
@@ -1,46 +1,77 @@
 import React, {Component} from 'react';
 import ReceiptInput from './ReceiptInput';
-import LineChart, {BarChart} from './UserChart';
-import axios from 'axios';
-//import "./App.css";
+import LineChart, {BarGraph, CategoryDoughnutChart, ItemDoughnutChart} from './UserChart';
 
+/**
+ * Corresponds to the different chart types.
+ */
+ const chart = {
```

Comment:
should we make a different component instead ? like dashboard or something. App.js seems to be clustered. We can do that in a future PR as well.

Reply:
Yea that sounds like a good idea